backend.native/tests/runtime/workers/worker11.kt

```diff
@@ -91,4 +91,27 @@ val counters = Array(COUNT) { AtomicInt(0) }.freeze()
   workers.forEach {
     assertFailsWith<IllegalStateException> { it.execute(TransferMode.SAFE, { Unit }) { println("ERROR") } }
   }
-}
\ No newline at end of file
+}
+
+@Test fun runTest2() {
+    val workers = Array(COUNT) { Worker.start() }
+    val futures = Array(workers.size) { workerIndex ->
+        workers[workerIndex].execute(TransferMode.SAFE, { null }) {
+            // Here we processed termination request.
+            assertEquals(false, Worker.current!!.processQueue())
```

Comment:
`processQueue` is non-blocking. Maybe wait until termination request sent?

Reply:
Here we either see termination request or see nothing, in both cases `false` to be returned. So your suggestion will make test more deterministic, but seems it's correct even in its current form.

Reply:
Exactly.